### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by EdAbati**
Smaller PR compared to #18.

- Removed unused files, variables etc
- Collated readmes and docs
- collated cli and notebooks in 1 location
- fixed some issues with model training and streamlit ui

I couldn't replicate the streamlit ui locally because I don't have access to S3 (but I tested it in my other PR)

We do we have fixed `image: ` names for `lambda` `jupiter` and `streamlit` in [here](https://github.com/EdAbati/fsdl-2022-weak-supervision-project/blob/clean-up-code/docker-compose.yml#L5)? I think it is not super handy if you need to change the Dockerfile during development.

### CODE REVIEW

**Comment by diegoquintanav** [comment]
Cool, this looks good to me. About the S3 credentials, I've left those in the lambdalabs machine. About the image names, it's precisely there for that: you can change the image names and their tags by overriding the env vars in your local `.env` file, which is not being tracked. Compose reads all environment variables automatically from that file. 

Thanks!

File: services/jupyter/src/pyproject.toml [hunk 1]
```diff
@@ -26,6 +26,7 @@ transformers = "^4.22.2"
 typer = "^0.6.1"
 wandb = "^0.13.3"
 boto3 = {version = "^1.24.89", source = "pypi"}
+python-dotenv = "^0.21.0"
```
Comment (diegoquintanav) [inline services/jupyter/src/pyproject.toml]: Pydantic's `BaseConfig` in `app/config.py` parses environment variables automatically, see https://pydantic-docs.helpmanual.io/usage/settings/. These are currently being passed to the container through docker-compose.

Comment (EdAbati) [reply services/jupyter/src/pyproject.toml]: changed back

**Comment by EdAbati** [comment]
Thanks @diegoquintanav for the quick review. Please see the last 4 commits, streamlit env should be a bit lighter now. and if you are happy we can merge

**Comment by diegoquintanav** [comment]
> Thanks @diegoquintanav for the quick review. Please see the last 4 commits, streamlit env should be a bit lighter now. and if you are happy we can merge

Looks good to me, thanks!

**Review by diegoquintanav — APPROVED** [review]

**Pull request merged**